articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md

```diff
@@ -0,0 +1,201 @@
+<!-- markdownlint-disable MD041 MD002 -->
+
+## Install the SDK
+
+You can install the [Auth0 Vue SDK](https://github.com/auth0/auth0-vue) using npm:
+
+```bash
+npm install @auth0/auth0-vue@beta
+```
+
+## Registering the plugin
```

Comment:
```suggestion
## Register the plugin
```

Reply:
Also, should this be a level 3 heading nested under "Install the SDK"? For me it's just a part of the installation.